### ISSUE TITLE
Decision to resurrect Nazgul

### ISSUE THREAD

**Issue by Helliaca** [Helliaca/HoI4-LotrMod#105]
since we know the nazguls cannot be killed, and well, kept on being brought back by sauron no matter what, i'd like to suggest an ability that if a nazgul dies in one of the many attempts to get the ring from the fellowship, as mordor you could bring them back, and welp, of course, you can't bring back the nazguls if the one ring is destroyed.

**Comment by Shooteger** [comment]
Nice idea, you think about making it a decision? Would suit better than a focus, or maybe a national focus just in mordor, which makes this decisions available and then have a cooldown of a year or so for balancing and keeping it "realistic".

**Comment by Helliaca** [comment]
Yeah, I think a decision with a hefty cost and cooldown is probably the way to go

**Comment by Shooteger** [comment]
> Yeah, I think a decision with a hefty cost and cooldown is probably the way to go

Maybe like add aditionally first, that you get (as mordor or any of the "bad" guys countries) a national focus with gives your troops a minus in moral, org and other things which would make sense, when a leader dies. Then start fixing that national focus by buying decisions for hefty PP and slowly fix it, with the last decision gives you back the leader any maybe even a hefty boost in moral and defense for a few days or months, when they get back.

This would make the player really want to get them back and can be seen as a "last stand".
